extensions/wikia/Wall/WallThread.class.php

```diff
@@ -77,31 +81,31 @@ private function loadReplyObjs() {
 		}
 	}
 
-	private function loadReplyIdsFromDB($master = false) {
+	/**
+	 * Fetches reply IDs for the thread, using a limit to control large queries
+	 *
+	 * @param DatabaseBase $dbr Database resource
+	 * @param integer $afterId The last reply ID after which the next set is selected
+	 * @return array List of reply IDs
+	 */
+	private function getReplyIdsFromDB( $dbr, $afterId = null ) {
```

Comment:
Limit and offset for use with paginating results are not good for performance, check out http://use-the-index-luke.com/no-offset